plugin/evm/block.go

```diff
@@ -136,13 +136,43 @@ func (b *Block) Accept() error {
 	if err != nil {
 		return err
 	}
-	for _, tx := range b.atomicTxs {
+
+	acceptAtomic := func(tx *Tx) error {
```

Comment:
New function for accepting the atomic transaction. Used for the two loops below because we didn't need to remove `atomicTransactor` transactions from the mempool